lib/binding_rust/build.rs

```diff
@@ -34,6 +37,44 @@ fn main() {
         .compile("tree-sitter");
 }
 
+#[cfg(feature = "bindgen")]
+fn generate_bindings() {
+    const HEADER_PATH: &'static str = "include/tree_sitter/api.h";
```

Comment:
Maybe move this outside the function but still guarded by the cfg? and 'static shouldn't be needed

Reply:
It's not needed anywhere else so I'd keep it closer to the point of use. Thanks for the `'static` note, old habbit.

Reply:
Yeah I wasn't sure if you can remove static in a non top-level scope :sweat_smile: